### DISCUSSION TITLE
Which input formats are supported by dftd4?

### DISCUSSION THREAD

**Discussion by sigbjobo** [dftd4/dftd4#170] [Q&A] [answered]
Hi, I tried to look through the documentation, but I cannot find which formats are supported by dftd4.

I am asking because I need to apply periodic boundary conditions for my calculations, and all examples are with XYZ, which does not contain box information. Could you clarify this?

All the best,
Sigbjørn

**Comment by awvwgk** [comment, marked as answer]
You should be able to supply coordinates in all supported formats described at https://grimme-lab.github.io/mctc-lib/page/index.html. For crystal inputs Vasp POSCARs, DFTB+ gen-format, Turbomole coords and FHI-aims geometry.in are suitable.
